cms/sass/components/_filters.scss

```diff
@@ -50,6 +50,8 @@
   max-height: $spacing-07;
   height: auto;
   overflow-y: auto;
+  //add minimal padding to ensure visible outline
+  padding-top: .2rem;
```

Comment:
To maintain style consistency, best to use the `$spacing-[XX]` SASS variables found in https://github.com/DOAJ/doaj/blob/develop/cms/sass/base/_spacing.scss

Reply:
Done in https://github.com/DOAJ/doaj/commit/186e2c3fc7989cd05f0911619fcd0055e1e12387